api/route/Article/articleRouter.ts

```diff
@@ -45,6 +47,15 @@ const articleRouter = createRouter('article', {
     method: 'get',
     handler: getAll,
   },
+  getAllImageGen: {
+    method: 'get',
+    handler: getAllImageGen,
+  },
+  recordImage: {
```

Comment:
@artem, this is probably another route that'll need a permission. We'll leave it for this PR though Greg.

Reply:
@ArtemKech At a later stage...

> We'll leave it for this PR though Greg 👍